front/app/containers/ProjectsShowPage/index.tsx

```diff
@@ -120,6 +136,18 @@ const ProjectsShowPage = memo<Props>(({ project, scrollToEventId }) => {
   } else if (projectNotFound) {
     content = <ProjectNotFound />;
   } else if (projectId && processType) {
+    let phaseParticipationMethod;
+
+    if (!isNilOrError(phases)) {
+      phaseParticipationMethod =
+        getLatestRelevantPhase(phases)?.attributes.participation_method;
```

Comment:
What will `getLatestRelevantPhase` return when an admin takes a survey from the back office? The participation method of the phase which is passed in the URL should be the method, shouldn't it?

Reply:
This is the live front office project page, so this will never have a phase ID in the URL right now.

I will update the onFormSubmission to also send a phaseId (if relevant):

```
    clHistory.push({
      pathname: `/projects/${_project?.attributes.slug}`,
      search: `?show_modal=true`,   <-- Add phaseId here
    });
```
@koendehondt I can't really test these cases since this branch doesn't have the working save for different phases - should I merge in your branch ([CL-1381-saving-phase-input](https://github.com/CitizenLabDotCo/citizenlab/tree/CL-1381-saving-phase-input))? It would mean I can't merge to master though.

Reply:
If it helps you, then please merge it to CL-1381-saving-phase-input.